src/content/docs/en/reference/modules/astro-config.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Config imports API Reference
+sidebar:
+  label: 'astro:config'
+i18nReady: true
+tableOfContents:
+  minHeadingLevel: 2
+  maxHeadingLevel: 6
+---
+
+import Since from '~/components/Since.astro';
+
+<p><Since v="5.7.0" /></p>
+
+This virtual module `astro:config` exposes a non-exhaustive, serializable, type-safe version of the Astro configuration. There are two submodules for accessing different subsets of your configuration values: [`/client`](#imports-from-astroconfigclient) and [`/server`](#imports-from-astroconfigserver).
+
+All available config values can be accessed from `astro:config/server`. However, for code executed on the client, only those values exposed by `astro:config/client` will be available.  This protects your information by only making some data available to the client.
+
+## Imports from `astro:config/client`
+
+```js
+import {
+  i18n, 
+  trailingSlash,
+  base,
+  build,
+  site,
+} from "astro:config/client"
```

Comment:
formatting nit
```suggestion
import {
  i18n, 
  trailingSlash,
  base,
  build,
  site
} from "astro:config/client";
```

Reply:
Yan, I updated your suggestion to put the comma back in after `site` because we *do* show trailing commas in all of these import listings on other pages. Now, it's just adding the semi-colon.

Reply:
I guess we might include that on other pages, but that's still something that would be removed by the user's formatter on JS and Astro files. Instead of an arbitrary rule for these listings, I'd rather just have the code look as closely formatted as possible if it doesn't compromise the readability. In this case, the line breaks on imports are a good compromise; the trailing comma, not really a win.

Super nitpicky? yeah, but that's kinda my thing 😆

Reply:
How should I resolve the comment?